zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/service/ObjectManagementService.kt

```diff
@@ -98,16 +104,174 @@ class ObjectManagementService(
+                        }
+                    } else {
+                        mapToObjectSearchParameter(
+                            searchField.key,
+                            searchField.fieldType,
+                            listOf(otherFilter.rangeFrom?.value, otherFilter.rangeTo?.value),
+                            searchField.dataType
+                        )
+                    }
+                }
+        }
+
+        val searchString = concatenateObjectSearchParameter(searchDtoList)
+
+        val objectTypePluginInstance = getObjectTypenApiPlugin(objectManagement.objecttypenApiPluginConfigurationId)
+
+        val objectenPluginInstance = getObjectenApiPlugin(objectManagement.objectenApiPluginConfigurationId)
+
+        val objectsList = objectenPluginInstance.getObjectsByObjectTypeIdWithSearchParams(
+            objectTypePluginInstance.url,
+            objectenPluginInstance.url,
+            objectManagement.objecttypeId,
+            searchString,
+            pageable
+        )
+
+        val objectsListDto = mapToObjectListRowDto(objectsList, id)
+
+        return PageImpl(objectsListDto, pageable, objectsList.count.toLong())
+    }
+
+    private fun mapToObjectListRowDto(
+        objectsList: ObjectsList,
+        objectManagementId: UUID
+    ): List<ObjectsListRowDto> {
+        val listColumns = searchListColumnService.findByOwnerId(objectManagementId.toString())
+        return objectsList.results.map {objects ->
+            val listRowDto = listColumns?.map {
+                listColumn ->
+                ObjectsListRowDto.ObjectsListItemDto(
+                    listColumn.key,
+                    objects.record.data?.at(listColumn.path)
+                )
+            }
+            ObjectsListRowDto(objects.uuid.toString(), listRowDto!!)
+        }
+    }
+
+    private fun mapToObjectSearchParameter(
+        key: String,
+        fieldType: FieldType,
+        value: Any,
+        dataType: DataType
+    ): List<ObjectSearchParameter> {
+        return when (fieldType) {
+            FieldType.TEXT_CONTAINS -> listOf(
+                ObjectSearchParameter(
+                    key,
+                    Comparator.STRING_CONTAINS,
+                    castValueToDataType(dataType, value)
+                )
+            )
+
+            FieldType.RANGE -> {
+                (value as List<Any?>).mapIndexed { index, rangeValue ->
+                    if (rangeValue != null) {
+                        ObjectSearchParameter(
+                            key,
+                            if(index == 0) Comparator.GREATER_THAN_OR_EQUAL_TO else Comparator.LOWER_THAN_OR_EQUAL_TO,
+                            castValueToDataType(dataType, rangeValue)
+                        )
+                    } else {
+                        null
+                    }
+                }.filterNotNull()
+            }
+
+            FieldType.MULTI_SELECT_DROPDOWN -> {
```

Comment:
Please throw an error here. Because unfortunately, the objectenAPI doesn't support 'OR' searches